### ISSUE TITLE
udx does not appear on Packagist

### ISSUE THREAD

**Issue by doubleforte** [udx/wp-stateless#654]
The `composer.json` file shows:

```
"require": {
    "php": ">=5.6.20",
    "composer/installers": "~1.0",
    "udx/lib-settings": "~0.3",
    "udx/lib-wp-bootstrap": "~1.3",
    "udx/lib-ud-api-client": "~1.2",
    "ccampbell/chromephp": "^4.1",
    "firebase/php-jwt": "^6.1.2",
    "wpmetabox/meta-box": "5.6.3"
  },
```

...but the udx packages are not available on Packagist. https://packagist.org/?query=udx

This is causing `composer install` to fail. I see the packages are already included in the `vendor/` dir, so if they are not required by composer, they should probably be removed?

This could result in Composer fetching a malicious package under the same name from Packagist. This misconfiguration could facilitate an attack against the build process of the application, known as dependency confusion.

**Comment by planv** [comment]
Hello @doubleforte 

Thank you. It uses our own Package Repositories for these libraries. I've updated `composer.json`. 
Will be released in version `3.2.3`

**Comment by planv** [comment]
Released. Closing the ticket.
Please let us know if you have any further questions
